baseballAPI.UnitTests/Services/PlayerServiceTests.cs

```diff
@@ -46,7 +47,7 @@ public void AssertGetPlayerIdReturnsId(People person)
         {
             var playerId = _service.GetPlayerId(person.NameFirst, person.NameLast);
 
-            Assert.That(playerId, Is.EqualTo(person.PlayerId));
+            Assert.That(playerId.ElementAt(0).PlayerId, Is.EqualTo(person.PlayerId));
```

Comment:
This is good to cover the cases where we only find one match.
Should probably add a test case to cover multiple matches being returned.